Dockerfile

```diff
@@ -0,0 +1,15 @@
+
+FROM python:3.12.3-alpine
+
+ENV PYTHONUNBUFFERED 1
+
+LABEL org.opencontainers.image.title="reqstool"
+LABEL org.opencontainers.image.description="This is custom Docker Image for reqstool."
+LABEL org.opencontainers.image.authors="sysdev@lfv.se"
```

Comment:
This email address is internal use mainly. Use info@lfv.se instead.

Reply:
Fixed now